files/calenstyle/update.json

```diff
@@ -0,0 +1,18 @@
+{
+  "packageManager": "github",
+  "name": "calenstyle",
+  "repo": "CuriousSolutions/CalenStyle",
+  "files": {
+    "basePath": "dist/",
+    "include": ["*.min.*", 
+                "calenstyle-jquery-ui-override.css", 
+                "calenstyle-iconfont.css", 
+                "fonts/calenstyle-iconfont.ttf", 
```

Comment:
To retain the directory structure you need to add `./` in the beginning.
For example `./fonts/calenstyle-iconfont.ttf`
Please update both PRs. Thanks